src/tests/ftest/launch.py

```diff
@@ -636,22 +658,31 @@ def get_nvme_replacement(args):
 
     # Get the list of NVMe PCI addresses found in the output
     output_str = "\n".join([line.decode("utf-8") for line in output_data[0][0]])
-    devices = find_pci_address(output_str)
+    if include_vmd_flag == "vmd":
+        devices = find_pci_address(output_str, False)
+    else:
+        devices = find_pci_address(output_str, True)
```

Comment:
No need to pass True as the default find_pci_address() option is True.

Reply:
Done.